### PR TITLE
Update mindsets.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Minor formatting changes.

### CODE REVIEW

**Comment by amyblais** [comment]
I'm assigning this to Carrie (Tech writer) but let me know if you need help.

File: company/about-mattermost/mindsets.md [hunk 1]
```diff
@@ -83,10 +81,10 @@ Using the analogy of changing lanes while driving a vehicle and learning to do a
 
 This process is similar to [Mini-boss, End-boss](https://docs.mattermost.com/process/training.html#mini-boss-end-boss), except that the mini-boss is also the new owner of a process, and not only a reviewer. Shoulder checks should be requested by new owners to avoid “crashing”:
 
-> * Making changes to systems that break existing processes and may lose data and hurt the productivity of others downstream without notice and without a replacement system in place \(behavior known as [“Dead Tarzan”](https://docs.mattermost.com/process/training.html#dead-tarzan)\).
-> * Repeatedly investing in mis-prioritized projects due to a misunderstanding of requirements from project stakeholders and insufficient confirmation of intended outcomes.
+* Making changes to systems that break existing processes and may lose data and hurt the productivity of others downstream without notice and without a replacement system in place \(behavior known as [“Dead Tarzan”](https://docs.mattermost.com/process/training.html#dead-tarzan)\).
```
Comment (cwarnermm) [inline company/about-mattermost/mindsets.md]:
```suggestion
* Making changes to systems that break existing processes may lose data and hurt the productivity of others downstream without notice and without a replacement system in place \(behavior known as [“Dead Tarzan”](https://docs.mattermost.com/process/training.html#dead-tarzan)\).
```

**Review by cwarnermm — CHANGES_REQUESTED** [review]
Very minor suggestion inline.

**Review by cwarnermm — APPROVED** [review]
Looks great! Thank you!

**Pull request merged**